projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/index.js

```diff
@@ -6,5 +6,9 @@ import AiAssistantPluginSidebar from './components/ai-assistant-plugin-sidebar';
 // plugin settings
 export const name = 'ai-assistant-plugin';
 export const settings = {
-	render: AiAssistantPluginSidebar,
+	render: () => (
+		<>
+			<AiAssistantPluginSidebar />
```

Comment:
I have tried rendering the `<OnboardingGuide />` here, it gets automatically wrapped in a `<div style="display:none;"></div>` and I am not sure how to get around that. I am guessing there is some setting somewhere where you need to define where you want components rendered in the blog post editor to show up, but I am also not sure about that because I figured rendering it in the already existing sidebar would work, but it did not